.eslintrc.js

```diff
@@ -6,6 +6,9 @@ module.exports = {
         "node": true,
         "es6": true
     },
+    "globals": {
+        BigInt: true,
+    },
```

Comment:
I think this will provide better typehinting:
```
    "lib": [
      "esnext.bigint"
    ],
```

Reply:
I think you mean that for the tsconfig which I already did, this is for eslint